### ISSUE TITLE
Break this into multiple repos?

### ISSUE THREAD

**Issue by gaand** [ga-wdi-boston/js-functions-ins-and-outs#12]
Ins, Out, and Closure?

See ga-wdi-boston/js-functions-ins-and-outs/issues/10

**Comment by jrhorn424** [comment]
Yes! Nice seams.

**Comment by jrhorn424** [comment]
Outs went pretty fast. I'd estimate about 15% of the total delivery time.

I'm not sure how high-priority this should be. I do think there's significant value in breaking this up, if for nothing else to clarify learning objectives for each part, and to give closures its own treatment. Breaking this up would also allow us to move closures to the third unit if we wished.

I still struggle to give examples of when closures are useful for web application solutions.

**Comment by gaand** [comment]
> I still struggle to give examples of when closures are useful for web application solutions.

Is that because we implicitly use IIFEs and so never pollute the global name space?

Maybe that's something we need to make explicit?

I'd like to refactor this part of the curriculum using some of the iceboxed repos I created last time through.  Breaking this up would be part of that process (or perhaps the start of that?).

**Comment by jrhorn424** [comment]
> > I still struggle to give examples of when closures are useful for web application solutions.
> 
> Is that because we implicitly use IIFEs and so never pollute the global name space?

Yes. Although, I don't think it occurred to me. The "data hiding" part of closures is clean to explain. The fact that the data is still available to inner functions, less so.

> I'd like to refactor this part of the curriculum...

:+1:
